specifyweb/frontend/js_src/lib/utils/parser/definitions.ts

```diff
@@ -286,7 +286,8 @@ export function resolveParser(
   const formatter =
     field.isRelationship === false ? field.getUiFormatter?.() : undefined;
   return mergeParsers(parser, {
-    pickListName: field.getPickList?.(),
+    pickListName:
+      field.name === 'age' ? 'systemGeologicPicklist' : field.getPickList?.(),
```

Comment:
Condition not needed after defining a CO -> Age picklist

```suggestion
    pickListName: field.getPickList?.(),
```